### PR TITLE
Prashant_Assignment_25_30mins

### PR DESCRIPTION
**Pull request by ghost**
Assignment for WAP to sum all the numbers in given string Done.
Assignment_25

### CODE REVIEW

File: src/prashantCJune20/SumofIntinString.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package prashantCJune20;
+
+public class SumofIntinString {
+	
+	void addDigits(String input) {
+		int addition = 0;
+		for(int index = 0;index<input.length();index++) {
+			if(Character.isDigit(input.charAt(index)))
+				addition = addition	+ Integer.parseInt(String.valueOf(input.charAt(index)));
```
Comment (ashishpawar517) [inline src/prashantCJune20/SumofIntinString.java]: Inside the If block instead this long statement 
```java 
addition = addition	+ Integer.parseInt(String.valueOf(input.charAt(index)));
```

You could simply do 
```java 
addition += input.charAt(index)-'0';
```

File: src/prashantCJune20/SumofIntinString.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package prashantCJune20;
+
+public class SumofIntinString {
+	
+	void addDigits(String input) {
+		int addition = 0;
+		for(int index = 0;index<input.length();index++) {
```
Comment (ashishpawar517) [inline src/prashantCJune20/SumofIntinString.java]: index of the  character isn't necessary for the execution you can simply just use `forEach` loop

File: src/prashantCJune20/SumofIntinString.java (hunk 1 above)
Comment (ghost) [reply src/prashantCJune20/SumofIntinString.java]: Logic changed as per comments.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Review by ashishpawar517 — COMMENTED** [review]
done !

**Pull request merged**